Assignment2/bgDani-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,43 @@
+{
+    "@context":{
+        "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+        "pre": "http://www.oeg-upm.net/Organization#",
+        "vcard":"http://www.w3.org/2006/vcards/ns#",
+        "sen": "http://www.oeg-upm.net/Organization/Class01/includes/Sensor029#",
+        "comp": "http://www.oeg-upm.net/Organization/Class01/includes/Computer101#",
+        "meas": "http://www.oeg-upm.net/Organization/Class01/includes/Sensor029/Measurement8401#",
+        "us": "http://www.oeg-upm.net/Organization/Class01/includes/Computer101/User10A#",
+        "sensor": "rdfs:label",
+        "computer": "rdfs:label",
+        "measurement":"rdfs:label",
+        "user":"rdfs:label",
+        "date":"vcard:rev",
+        "num":"rdfs:label",
+        "name":"rdfs:label"
+    },
+    "@id":"http://www.oeg-upm.net/Organization/Class01",
+    "@type":"pre:Class01/includes",
+    "sensor":"Sensor029",
+    "computer": "Computer101",
+    "Sensor029":{
+        "@id":"http://www.oeg-upm.net/Organization/Class01/Sensor029",
+        "@type":"sen:hasMeasurement",
+        "measurement":"Measurement8401"       
+    },
+    "Computer101":{
+        "@id":"http://www.oeg-upm.net/Organization/Class01/Computer101",
+        "@type":"comp:hasOwner",
+        "user":"User10A"
+    },
```

Comment:
You might define them into an array, e.g. "includes":[{Sensor029 attributes},{Computer101 attributes}]. Also, you have to define the "@type" correctly, because you are trying to define them as the relationships between Sensor and Measurement, and Computer and User. Check the "geometry" definition of the WiFi point on how to do it.